### DISCUSSION TITLE
Access to Developer Studio

### DISCUSSION THREAD

**Discussion by emad-siddiq** [Fiserv/Support#86] [Q&A]
Hello, 
I was wondering what are the steps required to get access to a Sandbox environment? I am trying to get access to try the API's as a developer but the Sign Up link doesn't appear in the top right corner, as the documentation suggests.
Thanks, 
Emad Siddiq

**Comment by TaniaS-fiserv** [comment]
Hi @adriatic1517 Thank you for reaching out to us!

Please let us know which product on Developer Studio are you looking to integrate with ? We haven't rolled out the Sign Up feature in production yet!. 

Also what company/product do you work for? This would help us redirect you to the appropriate team to handle any questions you might have in future.

Thanks!

**Comment by BRENLOVE** [comment]
is anyone else having trouble opening emails from fiserv dev tech department
